src/main/java/com/greencity/ui/page/EcoNewsPage.java

```diff
@@ -0,0 +1,36 @@
+package com.greencity.ui.page;
+import lombok.Getter;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class EcoNewsPage extends BasePage {
+//    protected ToolbarComponent toolbarComponent;
+//    protected EcoNewsCardContainer ecoNewsCardContainer;
```

Comment:
Commented code should not be included in the main branch, remove it.
This functionality will be added when it is implemented.